dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hdfs/HdfsPermissionExtractionTask.java

```diff
@@ -68,7 +69,7 @@ public Void run(TaskRunContext context) throws Exception {
     return null;
   }
 
-  private void doRun(Writer output) throws IOException {
+  private void doRun(Writer output) throws IOException, ExecutionException, InterruptedException {
```

Comment:
I recommend adding tests for this method

Reply:
You have a point here, but let me get this PR through first.